internal/service/cloudbackupsnapshotexportbucket/resource_cloud_backup_snapshot_export_bucket.go

```diff
@@ -73,10 +72,9 @@ func Schema() map[string]*schema.Schema {
 			ForceNew: true,
 		},
 		"tenant_id": {
-			Deprecated: fmt.Sprintf(constant.DeprecationParamByVersion, "1.27.0") + " This field is ignored; the `mongodbatlas_cloud_provider_access_authorization.azure.tenant_id` is used instead.",
-			Type:       schema.TypeString,
-			Optional:   true,
-			Computed:   true,
+			Type:     schema.TypeString,
+			Optional: true,
```

Comment:
we could create a ticket to remove this in the future
```suggestion
			Optional: true, // attribute is only used as a computed, this is called out in docs and configuration of optional argument can be eventually removed implying a breaking change
```

Reply:
Thanks. Changed in 21af3496940a8e325017c5461b0ba21fe9e4d557
Created a ticket https://jira.mongodb.org/browse/CLOUDP-293142